README.rst

```diff
@@ -148,16 +148,20 @@ the administration dashboard.
 Connection App
 ~~~~~~~~~~~~~~
 
-This app enables the controller to instantiate connections to the devices
+This app enables to controller to utilize various protocols for designated purposes.
```

Comment:
This app enables to controller, why did you change `the` to `to`?

Reply:
Sorry this is a typo, I will fix this